HW0/P4/P4.py

```diff
@@ -40,37 +116,145 @@
     # Read the observation array and plot it (Part 2)
     #####################
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    # Alias the inverses for clarity
+    rx_inv = 1. / rx
+    ry_inv = 1. / ry
+    rz_inv = 1. / rz
+
+    # Get an ndarray
+    inv_mat = np.array([[rx_inv, 0, 0],
+                        [0, ry_inv, 0],
+                        [0, 0, rz_inv]])
+
+    # Load in the measured data
+    m_data = np.loadtxt('P4_measurements.txt', delimiter=',')
+
+    # Separate the data
+    m_x = m_data[:, 0]
+    m_y = m_data[:, 1]
+    m_z = m_data[:, 2]
```

Comment:
Actually, you can't - they'll all point to the same place in memory, and modifying one of them will modify all of the others.